firestore-send-email/integration-test/firebase.json

```diff
@@ -1,24 +1,23 @@
 {
   "extensions": {
-    "firestore-send-email": "firebase/storage-resize-images@0.1.27"
+    "firestore-send-email": "../"
   },
-
   "storage": {
     "rules": "storage.rules"
   },
-  "functions": { "runtime": "nodejs14", "source": "functions" },
   "emulators": {
     "hub": {
       "port": 4000
     },
     "storage": {
       "port": 9199
     },
-    "functions": {
-      "port": 9002
-    },
     "ui": {
       "enabled": true
     }
+  },
+  "firestore": {
```

Comment:
This extension uses firestore, so I needed to add this block + rules files so that the firestore emulator would run.